services/QuillLMS/app/models/classroom.rb

```diff
@@ -59,6 +59,12 @@ class Classroom < ApplicationRecord
 
   accepts_nested_attributes_for :classrooms_teachers
 
+  def destroy 
```

Comment:
Delete the join records, but don't delete units, activities, and other records that could be owned by other entities.